addons/osfstorage/models.py

```diff
@@ -318,19 +318,6 @@ def create_version(self, creator, location, metadata=None):
 
         return version
 
-    def get_version(self, version=None, required=False):
```

Comment:
Instead of modifying both `get_version` functions I just consolidated them into one function.